Operations_C/main.c

```diff
@@ -39,17 +40,93 @@ int Compare(int a, int b)
     else return -1;
 }
 
+void MoveBites()
```

Comment:
I like your new names conventions. It is much more understandable than just FirstMethod and SecondMethod. No need for an action here. Just resolve comment after you read it.